### PR TITLE
Adds message generic event to robots and devices

### PR DESCRIPTION
**Pull request by edgarsilva**

### CODE REVIEW

File: examples/generic_message_event/blink-client.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html>
+  <meta charset="utf-8">
+  <head>
+    <title>Socket.IO chat</title>
```
Comment (stewart) [inline examples/generic_message_event/blink-client.html]: Shouldn't this be renamed?

**Comment by edgarsilva** [comment]
I think so since it is no longer a chat of any kind. updating.

File: lib/socket-master.js [hunk 2]
```diff
@@ -109,44 +109,71 @@ SocketMaster.prototype.socketDevices = function(robot) {
...
-    item.on(eventName, function() {
-      var args = _.toArray(arguments),
-          nsp = this.nsp[name];
-
-      args.unshift(eventName);
-
-      nsp.emit.apply(nsp, args);
-    }.bind(this));
+    console.log(item.listeners(eventName));
+    console.log('item  ===>>', item);
```
Comment (stewart) [inline lib/socket-master.js]: Is this logging supposed to be here?

**Comment by edgarsilva** [comment]
Should be ready to rumble now.

**Comment by stewart** [comment]
:8ball:

**Pull request merged**